app/uk/gov/hmrc/individualsmatchingapi/controllers/v2/PrivilegedCitizenMatchingController.scala

```diff
@@ -66,7 +66,10 @@ class PrivilegedCitizenMatchingController @Inject()(
           Ok(response)
         }
       }
-    } recover recoveryWithAudit(maybeCorrelationId(request), request.body.toString, "/individuals/matching/")
+    } recover recoveryWithAudit(
```

Comment:
if this is auditing then do we need to redact?

Reply:
I believe when there is an exception, we use this to log in kibana - ends up potentially logging PII

Reply:
maybe we should change the name of the function to `recoveryWithLogging` ?

Reply:
done 😄